tools/quantize/quantize.cpp

```diff
@@ -1,5 +1,6 @@
 #include "common.h"
 #include "llama.h"
+#include "llama-quant.h"
```

Comment:
We should avoid including private headers.

Reply:
I'll update as suggested